neo/rawio/neuralynxrawio.py

```diff
@@ -221,8 +244,9 @@ def _parse_header(self):
                     ts0 = ts[0]
                     ts1 = ts[-1]
                 ts0 = min(ts0, ts[0])
-                ts1 = max(ts0, ts[-1])
+                ts1 = max(ts0, ts[-1])  # :FIXME: possible error, should be ts1
```

Comment:
Yes, I think this is indeed wrong and should read `ts1 = max(ts1, ts[-1])` instead.

Reply:
I fixed the this particular bug and removed the :FIXME: comment. The :TODO: comments can remain for now and will be partially addressed in a new PR soon.